packages/near-contract-standards/src/fungible_token/core_impl.ts

```diff
@@ -123,7 +123,7 @@ export class FungibleToken implements FungibleTokenCore, StorageManagement, Fung
         // Get the unused amount from the `ft_on_transfer` call result.
         let unused_amount: Balance;
         try {
-            const promise_result = near.promiseResult(0).replace(/"*/g, ''); //TODO: why promiseResult returnes result with brackets?
+            const promise_result = near.promiseResult(0).replace(/"*/g, ''); //TODO: why promiseResult returns result with brackets?
```

Comment:
[FT standard](https://github.com/near/NEPs/blob/master/neps/nep-0141.md#ft_on_transfer) defined the encoding for the input/outputs as JSON. Since JSON does not support big integers, the FT standard uses a string type to pass the unused_amount. It would be cleaner to use `JSON.parse()` instead of manually "parsing" quotes out.

Reply:
@frol, this is out of this PR's scope, would be better in a separate PR